doc/audio.md

```diff
@@ -1,8 +1,17 @@
 # Audio
 
-To play an audio is really simple!
+To play a audio is really simple!
```

Comment:
https://www.a-or-an.com/a_an/audio

Reply:
Starting with To is a bit uncommon to me, I think `Playing an audio...` sounds better?

Reply:
Ah, I started rewriting this sentence, but forgot to finish it!
Fixed it now